src/javascript/app/pages/user/account/settings/two_factor_authentication.js

```diff
@@ -0,0 +1,109 @@
+const QRCode           = require('davidshimjs-qrcodejs');
+const BinarySocket     = require('../../../../base/socket');
+const Client           = require('../../../../base/client');
+const FormManager      = require('../../../../common/form_manager');
+const getPropertyValue = require('../../../../../_common/utility').getPropertyValue;
+const localize         = require('../../../../../_common/localize').localize;
+
+const TwoFactorAuthentication = (() => {
+    const form_id = '#frm_two_factor_auth';
+    let enabled_state,
+        next_state,
+        qrcode; // eslint-disable-line
+
+    const onLoad = () => {
+        init();
+    };
+
+    const init = () => {
+        BinarySocket.send({ account_security: 1, totp_action: 'status'}).then((res) => {
+            $('#two_factor_loading').remove();
+
+            if (res.error) {
+                handleError('status', res.error.message);
+                return;
+            }
+
+            enabled_state = res.account_security.totp.is_enabled ? 'enabled' : 'disabled';
+            next_state = res.account_security.totp.is_enabled ? 'disable' : 'enable';
```

Comment:
can you align by `=`?

Reply:
we can keep 'enabled' and 'disabled' in an array and use +(!is_enabled) and +is_enabled alternatively to get the values for enabled state and next state